packages/core/core/src/requests/PathRequest.js

```diff
@@ -280,14 +288,16 @@ export class ResolverRunner {
 
     let specifier = escapeMarkdown(dependency.moduleSpecifier || '');
 
-    // $FlowFixMe because of the err.code assignment
-    let err = await this.getThrowableDiagnostic(
-      dependency,
-      md`Failed to resolve '${specifier}' ${dir ? `from '${dir}'` : ''}`,
-    );
+    let err =
+      diagnostics.length > 0
+        ? new ThrowableDiagnostic({diagnostic: diagnostics})
+        : // $FlowFixMe because of the err.code assignment
+          await this.getThrowableDiagnostic(
+            dependency,
+            md`Failed to resolve '${specifier}' ${dir ? `from '${dir}'` : ''}`,
+          );
```

Comment:
Changed this to avoid a double error appearing whenever a resolver throws an error. I don't think we need to show "Failed to resolve" in addition to the diagnostic thrown by the resolver, right?